src/app.js

```diff
@@ -13,16 +13,26 @@ const APP_VIEWS = [
 app.set("view engine", configureNunjucks(app, APP_VIEWS));
 app.use(
   "/assets",
-  express.static(
-    path.join(__dirname, "/node_modules/govuk-frontend/govuk/assets"),
-  ),
+  express.static(path.join(__dirname, "/node_modules/govuk-frontend/govuk/assets"))
 );
 app.use(express.static("public"));
 
 app.get("/", (req, res) => {
   res.render("index.njk");
 });
 
+app.get("/service-description", (req, res) => {
```

Comment:
@SophiaDWokoma fyi this is where the new routes are :)

Reply:
Thank youu